dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageRenderUtil.java

```diff
@@ -286,19 +331,63 @@ private List<ContainerRaw> populateContainers() throws DotDataException, DotSecu
         return raws;
     }
 
+    /**
+     * When <b>in Edit Mode only</b>, we need to read the number of Containers in any HTML Page in the repository that
+     * might be referencing a given Contentlet. This piece of information will be added to the Contentlet's map so that
+     * the UI can let the editing User choose whether they want to edit just that piece of Content, or if they want to
+     * edit the "global" content. This new property is specified in {@link Contentlet#ON_NUMBER_OF_PAGES}.
+     *
+     * @param contentlet The {@link Contentlet} whose HTML Page references will be counted.
+     */
+    private void addContentletPageReferenceCount(final Contentlet contentlet) {
+        if (this.mode.equals(PageMode.EDIT_MODE)) {
```

Comment:
this would be nicer such as isEditMode()

Reply:
Done!